packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -4060,9 +4060,12 @@ export type StatefulCallClientOptions = {
 
 // @public
 export interface StatefulChatClient extends ChatClient {
+    dispose(): void;
```

Comment:
Should the name be more specific? Seems like the function is tied to cache.

Reply:
I think its fine to keep it dispose(). Its a pretty standard naming convention to use when cleaning things up. In future we can also clean up the chat threads as well in this method.